odl/discr/diff_ops.py

```diff
@@ -87,7 +87,7 @@ def __init__(self, space, axis=0, method='forward', padding_method=None,
             accuracy in the interior.
         """
         if not isinstance(space, DiscreteLp):
-            raise TypeError('space {!r} is not a DiscreteLp instance.'
+            raise TypeError('`space` {!r} is not a DiscreteLp instance.'
```

Comment:
That implies that we *have to* rewrite the error messages for subclasses which use a different parameter name than their parents. We'll get some duplicate checks there. We can decide to do it that way.